src/common/ms-word-helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { OutputData as RichTextData } from '@editorjs/editorjs';
+import { FieldInputProps } from 'react-final-form';
+import { Nullable } from './types';
+
+export const handleMsUnorderedList = (
+  event: ClipboardEvent,
+  input: FieldInputProps<Nullable<RichTextData>>
+) => {
+  event.preventDefault();
+  const text: string | undefined = event.clipboardData?.getData('text');
+
+  // Match any line that starts with "•\t" followed by anything until the end of the line
+  const matches: RegExpMatchArray | null | undefined = text?.match(/•\t(.*)/g);
+
+  if (matches) {
+    // If there are matches, create an array of strings without the "•\t" prefix
+    const listItems: string[] = matches.map((item) => item.replace(/•\t/, ''));
```

Comment:
In the cases where the unordered list uses dashed or custom bullet points in the list,  will this regex catch that?

Reply:
see other answer :)